src/index.js

```diff
@@ -3,13 +3,28 @@ import { useState } from 'react';
 import usePersistedState from './usePersistedState';
 import createStorage from './createStorage';
 
-const createPersistedState = (
-  key,
-  provider = global.localStorage
-) => {
+const getProvider = () => {
+  if (typeof global !== 'undefined' && global.localStorage) {
+    return global.localStorage;
+  }
+  // eslint-disable-next-line no-undef
+  if (typeof globalThis !== 'undefined' && globalThis.localStorage) {
+    // eslint-disable-next-line no-undef
```

Comment:
Seems yes, didn't add this myself but if I remove the comment it is showing a message for `globalThis`, I guess we could configure it in the eslint config but since we rarely change this library will just keep it

Reply:
Seems yes, didn't add this myself but if I remove the comment it is showing a message for `globalThis`, I guess we could configure it in the eslint config but since we rarely change this library I think I will just keep it to have a smaller diff

Reply:
Seems yes, didn't add this myself but if I remove the comment it is showing a message for `globalThis`, I guess we could configure it in the eslint config but since we rarely change this library so I think I will just keep it to have a smaller diff